### ISSUE TITLE
pip install failed

### ISSUE THREAD

**Issue by giswqs** [UnfoldedInc/earthengine-layers#26]
I tried to install [pydeck-earthengine-layers](https://pypi.org/project/pydeck-earthengine-layers/) using pip, but it failed. 

> pip install pydeck-earthengine-layers
Collecting pydeck-earthengine-layers
  Downloading pydeck-earthengine-layers-0.1.0.tar.gz (3.9 kB)
    ERROR: Command errored out with exit status 1:
     command: /home/qiusheng/.conda/envs/geemap/bin/python3.8 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ujzobw6a/pydeck-earthengine-layers/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ujzobw6a/pydeck-earthengine-layers/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-ujzobw6a/pydeck-earthengine-layers/pip-egg-info
         cwd: /tmp/pip-install-ujzobw6a/pydeck-earthengine-layers/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-ujzobw6a/pydeck-earthengine-layers/setup.py", line 13, in <module>
        with open('requirements.txt') as requirements_file:
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

**Comment by kylebarron** [comment]
Thanks for reporting. I appear to have missed `requirements.txt` in the [`MANIFEST.in`](https://github.com/UnfoldedInc/earthengine-layers/pull/24/files#diff-480f8b9e96e23cd5daeefc0848539d6fR1-R4). I'll fix this quickly and ping you when a new version is up.

**Comment by giswqs** [comment]
Also the hyperlink (https://pypi.org/project/pydeck-earthengine-layers/examples/Introduction.ipynb) for the `examples/Introduction.ipynb` on the pypi page is broken. The correct link should be https://github.com/UnfoldedInc/earthengine-layers/blob/master/examples/pydeck/Introduction.ipynb
```
Using
For an example, see the examples/Introduction.ipynb Jupyter Notebook.
```

**Comment by kylebarron** [comment]
I just pushed a patch, and it's up on PyPI as 0.1.1. I tested installing from pip in a new environment, as described [here](https://nbviewer.jupyter.org/github/kylebarron/earthengine-layers/blob/python-wrapper/py/examples/Introduction.ipynb), and it worked for me. Let me know if you still have an issue installing.

And yes, you're right. That link works on Github, but not on PyPI because it doesn't understand relative links. I'll move that link to an absolute link once the notebook path is stable. E.g. that notebook listed above is the older version of the notebook in [this PR](https://github.com/UnfoldedInc/earthengine-layers/pull/24/files#diff-37374b85e2e80e5055aa3fb3ba20dc17) (which is what I linked above). This notebook uses the `pydeck-earthengine-layers` pip module.
